h5p-bildetema/src/components/SearchPage/SearchView/SearchView.tsx

```diff
@@ -61,6 +73,16 @@ const SearchView = ({
             variant="secondary"
             labelPrefix={searchLabel}
           />
+          <Select
+            options={languages}
+            handleChange={handleViewLanguageChange}
```

Comment:
Make sure you keep this pattern like : handleChange{handleViewLanguageChange} instead of   handleChange{  handleChange}